searx/results.py

```diff
@@ -355,7 +355,7 @@ def close(self):
 
         for result in self._merged_results:
             score = result_score(result)
-            result['score'] = score
+            result['score'] = result.get('score', 0) + score
```

Comment:
The score is calculated using the `weight` factor of the engine, I'm not sure if an constant value subtracted (or added) from (to) the score is the best choice.

The factor `weight` has an exponential effect, e.g. if several engines have this result item ...

https://github.com/searxng/searxng/blob/75e4b65127668a1d3bfe559ecca38687b6433a7c/searx/results.py#L133-L142

The deduction of a constant from the score only has a linear effect ... and with increasing `weight` the effect of this constant on the result becomes less and less.

But since the calculation is encapsulated behind `low_priority` and `high_priority` we can also think about the calculation and weighting in another PR / for me this solution would be acceptable for now.